node/src/chain_spec/rococo.rs

```diff
@@ -0,0 +1,129 @@
+
+	Ok(ChainSpec::from_genesis(
+		// Name
+		"HydraDX testnet",
+		// ID
+		"hydra_rococo",
+		ChainType::Live,
+		move || {
+			parachain_genesis(
+				wasm_binary,
+				// Sudo account
+				// Galactic Council
+				// 7JcAAB6cXQxVQyVLksPUdthJwcoEGm8SW9hsNgdP6hjme5J1
+				hex!["2cb1a0ef4ce819893905e3a6a8e46b652c43aee6c154921220902cabfdcfdd07"].into(),
+				// initial authorities & invulnerable collators
+				(
+					vec![
+						(
+							// 5DvFqJq182asuR9EKoBBJBGEnZwpMHvuBrYpiMdr4hS8B6Eh
+							hex!["5206e6a18c96bab98f459ab636226481699220cf94346d766cd1142557a2fc66"].into(),
+							hex!["5206e6a18c96bab98f459ab636226481699220cf94346d766cd1142557a2fc66"].unchecked_into(),
+						),
+						(
+							// 5Fbc5bQp1bHfNpx8yuTshYK1pNhBbajZyaosRNvuyM3KRiMz
+							hex!["9c45dc3b15cd55531cad1e4c21cacb47611be54e3da6bf5be451f5e578a68344"].into(),
+							hex!["9c45dc3b15cd55531cad1e4c21cacb47611be54e3da6bf5be451f5e578a68344"].unchecked_into(),
+						),
+						(
+							// 5CAP1unPwP9RnvNcJn7YSZy1A8Snv5fGebuWMU99vpaJcfjh
+							hex!["04540c9406af2f5bf34f948dd1c7f029892247c2e1472b7cd51c188e4e0c2f2b"].into(),
+							hex!["04540c9406af2f5bf34f948dd1c7f029892247c2e1472b7cd51c188e4e0c2f2b"].unchecked_into(),
+						),
+						(
+							// 5GrzdLdMFuV6ZvWQjYUhVEpZQFoSxkLboKY3xeEdEjNUPoH7
+							hex!["d43ead05edb218c199fc0dfe36ec9389f509d8447e43109e1cb04305de4f9359"].into(),
+							hex!["d43ead05edb218c199fc0dfe36ec9389f509d8447e43109e1cb04305de4f9359"].unchecked_into(),
+						),
+						(
+							// 5G9gRv1GE8HPsjXnNm8By2SWuasTan5gU2AZm8pundyduHTM
+							hex!["b4bc5b99d9207ab2aadd222581809b53372e7e17c4eaa88742f3501f3044bf27"].into(),
+							hex!["b4bc5b99d9207ab2aadd222581809b53372e7e17c4eaa88742f3501f3044bf27"].unchecked_into(),
+						),
+					],
+					10_000 * UNITS,
+				),
+				// Pre-funded accounts
+				vec![(
+					// Galactic Council
+					// 7JcAAB6cXQxVQyVLksPUdthJwcoEGm8SW9hsNgdP6hjme5J1
+					hex!["2cb1a0ef4ce819893905e3a6a8e46b652c43aee6c154921220902cabfdcfdd07"].into(),
+					1_500_000_000,
+				)],
+				// council members
+				// GC - same as sudo
+				vec![hex!["2cb1a0ef4ce819893905e3a6a8e46b652c43aee6c154921220902cabfdcfdd07"].into()],
+				// technical committee
+				// GC - same as sudo
+				vec![hex!["2cb1a0ef4ce819893905e3a6a8e46b652c43aee6c154921220902cabfdcfdd07"].into()],
+				// vestings
+				vec![],
+				// registered_assets
+				vec![],
+				// accepted_assets
+				vec![],
+				// tx_fee_payment_account
+				// treasury
+				hex!["6d6f646c70792f74727372790000000000000000000000000000000000000000"].into(),
+				// token balances
+				vec![],
+				// claims data
+				Default::default(),
+				// elections
+				vec![(
+					hex!["2cb1a0ef4ce819893905e3a6a8e46b652c43aee6c154921220902cabfdcfdd07"].into(),
+					1_200_000_000 * UNITS,
+				)],
+				// parachain ID
+				PARA_ID.into(),
+			)
+		},
+		// Bootnodes
+		vec![
+			"/dns/p2p-01.hydra.hydradx.io/tcp/30333/p2p/12D3KooWCtBQpwnWV7yMaEyBRkcAcAej78Q2uZawk5RcDrYktVQS"
```

Comment:
Could we maybe avoid using the legacy hydradx.io domain ?

Reply:
what is the bootnode url then?